### PR TITLE
Creation of content layout, footer and navbar of the page

### PR DESCRIPTION
**Pull request by jeymarsonwolox**
**Summary**
Add items specified in the cards with their respective scss.

**Screenshots**
cap-001
cap-002

**Trello Card**
https://trello.com/c/BRjY95n0/11-setup-del-proyecto

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,14 +1,159 @@
...
-  <h1 class="title">Hello World</h1> 
+  <div class="nav">
+    <img src="assets/wolox_logo.svg" alt="Wolox">
+    <p class="nav-title">
+      Developer Tools
+    </p>
+  </div>
+  <div class="main">
+    <div class="div-title">
+      <img src="assets/triangle.svg" alt="Front-End Tools" class="img-title">
```
Comment (FrankIglesias) [inline index.html]: You can change this to a pseudo element

File: index.html [hunk 2]
```diff
@@ -1,14 +1,159 @@
...
+      <h4 class="text-title">Front-End Tools</h4>
```
Comment (FrankIglesias) [inline index.html]: change this for an h2

File: index.html [hunk 3]
```diff
@@ -1,14 +1,159 @@
...
+    </div>
+    <div class="div-cont-tech">
+      <div class="div-tech">
+        <div class="tech-back"></div>
```
Comment (FrankIglesias) [inline index.html]: this can be a pseudo element

File: index.html [hunk 4]
```diff
@@ -1,14 +1,159 @@
...
+    <p class="nav-title">
+      Developer Tools
+    </p>
+  </div>
+  <div class="main">
+    <div class="div-title">
+      <img src="assets/triangle.svg" alt="Front-End Tools" class="img-title">
+      <h4 class="text-title">Front-End Tools</h4>
+    </div>
+    <div class="div-cont-tech">
```
Comment (FrankIglesias) [inline index.html]: and by doing that you can remove this

File: index.html [hunk 5]
```diff
@@ -1,14 +1,159 @@
...
+      <div class="div-tech">
```
Comment (FrankIglesias) [inline index.html]: use another name for this class

File: index.html [hunk 6]
```diff
@@ -1,14 +1,159 @@
...
+        <div class="tech-back"></div>
+        <div class="tech-content">
+          <img src="assets/react_icon.svg" alt="ReactJS" class="img-tech">
```
Comment (FrankIglesias) [inline index.html]: add a slash before >

File: index.html [hunk 7]
```diff
@@ -1,14 +1,159 @@
...
+          <div class="div-info-tech">
+            <p class="tech-title">ReactJS</p>
```
Comment (FrankIglesias) [inline index.html]: use a more important tag like h3 or h4

File: index.html [hunk 8]
```diff
@@ -1,14 +1,159 @@
...
+            <br>
```
Comment (FrankIglesias) [inline index.html]: delete this

File: index.html [hunk 9]
```diff
@@ -1,14 +1,159 @@
...
+    <div class="div-title">
+      <img src="assets/triangle.svg" alt="Front-End Tools" class="img-title">
+      <h4 class="text-title">Front-End Tools</h4>
+    </div>
+    <div class="div-cont-tech">
+      <div class="div-tech">
+        <div class="tech-back"></div>
+        <div class="tech-content">
+          <img src="assets/react_icon.svg" alt="ReactJS" class="img-tech">
+          <div class="div-info-tech">
```
Comment (FrankIglesias) [inline index.html]: delete this

File: index.html [hunk 10]
```diff
@@ -1,14 +1,159 @@
...
+            <p class="tech-title">ReactJS</p>
+            <br>
+            <p class="tech-release">Release: 2013</p>
+          </div>
+          <p class="tech-description">React is a declarative, component-based Javascript library for building user
+            interfaces.</p>
+        </div>
+      </div>
+      <div class="div-tech">
```
Comment (FrankIglesias) [inline index.html]: Do the same for all techs

File: index.html [hunk 11]
```diff
@@ -1,14 +1,159 @@
...
+            <br>
+            <p class="tech-release">Release: 2005</p>
+          </div>
+          <p class="tech-description">Django (/ˈdʒæŋɡoʊ/ JANG-goh)[5] is a free and open-source web framework, written
+            in Python, which follows the model-view-template (MVT) architectural pattern.</p>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="footer">
```
Comment (FrankIglesias) [inline index.html]: use a footer here

File: index.html [hunk 12]
```diff
@@ -1,14 +1,159 @@
...
 <body>
-  <h1 class="title">Hello World</h1> 
+  <div class="nav">
+    <img src="assets/wolox_logo.svg" alt="Wolox">
+    <p class="nav-title">
+      Developer Tools
+    </p>
+  </div>
+  <div class="main">
+    <div class="div-title">
```
Comment (FrankIglesias) [inline index.html]: delete this div

File: index.html [hunk 13]
```diff
@@ -1,14 +1,159 @@
...
   <meta name="description" content="training">
   <meta name="author" content="Wolox">
-  <link rel="stylesheet"  type="text/css" href="application.css">
+  <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
+  <link rel="stylesheet" type="text/css" href="application.css">
 </head>
+
 <body>
-  <h1 class="title">Hello World</h1> 
+  <div class="nav">
```
Comment (FrankIglesias) [inline index.html]: use a nav tag

File: index.html [hunk 14]
```diff
@@ -1,14 +1,159 @@
...
 <head>
   <meta charset="utf-8">
   <title>HTML & CSS Training</title>
   <meta name="description" content="training">
   <meta name="author" content="Wolox">
-  <link rel="stylesheet"  type="text/css" href="application.css">
+  <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
+  <link rel="stylesheet" type="text/css" href="application.css">
 </head>
+
```
Comment (FrankIglesias) [inline index.html]: delete this line

File: scss/footer.scss [hunk 15]
```diff
@@ -0,0 +1,28 @@
+@import 'colors';
+@import 'text';
+
+$text-color: white;
```
Comment (FrankIglesias) [inline scss/footer.scss]: move this to colors

File: scss/navbar.scss [hunk 16]
```diff
@@ -0,0 +1,22 @@
+@import 'text';
+@import 'colors';
+
+$box-shadow: 0 3px 10px $grey;
+
+.nav {
+  align-items: center;
+  background-color: white;
```
Comment (FrankIglesias) [inline scss/navbar.scss]: add a color varaible

File: scss/navbar.scss [hunk 17]
```diff
@@ -0,0 +1,22 @@
...
+  box-shadow: $box-shadow;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  padding: 20px 10px;
+  position: fixed;
+  top: 0;
```
Comment (FrankIglesias) [inline scss/navbar.scss:16]: add a left prop

File: scss/tech.scss [hunk 18]
```diff
@@ -0,0 +1,69 @@
...
+  justify-content: space-between;
+  min-width: $width;
+  padding: 10px 40px 35px 25px;
+  position: relative;
+  right: ($width - 8);
+  text-align: center;
+  top: 8px;
+}
+
+.div-tech:hover > .tech-back {
```
Comment (FrankIglesias) [inline scss/tech.scss]: nest this rule inside the .div-tech class

File: scss/titles.scss [hunk 19]
```diff
@@ -0,0 +1,18 @@
...
+
+.img-title {
+  height: 90px;
+  width: 70px;
+}
+
+.text-title {
+  font-size: $text-big;
+  position: relative;
+  right: 18px;
```
Comment (FrankIglesias) [inline scss/titles.scss]: why this right and top?

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

**Comment by jeymarsonwolox** [comment]
Corrections are made to annotations of the pull request.

File: scss/titles.scss [hunk 20]
```diff
@@ -0,0 +1,21 @@
+@import 'text';
+
+.title::before {
```
Comment (FrankIglesias) [inline scss/titles.scss]: You can nest this pseudo element inside the title class

File: index.html [hunk 21]
```diff
@@ -1,14 +1,109 @@
...
+    <div class="tech">
+      <div class="tech-content">
+        <img src="assets/react_icon.svg" alt="ReactJS" class="img-tech" />
+        <h4 class="tech-title">ReactJS</h4>
+        <p class="tech-release">Release: 2013</p>
+        <p class="tech-description">React is a declarative, component-based Javascript library for building user
+          interfaces.</p>
+      </div>
+    </div>
+    <div class="tech">
```
Comment (FrankIglesias) [inline index.html:27]: delete this div and ony use one div

File: scss/tech.scss [hunk 22]
```diff
@@ -0,0 +1,72 @@
...
+  background-color: $green;
+  border-radius: $border-radius;
+  display: flex;
+  flex-direction: column;
+  height: $height;
+  justify-content: space-between;
+  min-width: $width;
+  padding: 10px 40px 35px 25px;
+  position: relative;
+  right: ($width - 8);
```
Comment (FrankIglesias) [inline scss/tech.scss]: 8px

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

**Comment by jeymarsonwolox** [comment]
Corrections are made to annotations of the pull request.

File: scss/footer.scss [hunk 23]
```diff
@@ -0,0 +1,26 @@
+@import 'colors';
+@import 'text';
+
+.footer {
+  align-items: center;
+  background-color: $blue;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  padding: 45px 25px 15px 25px;
```
Comment (FrankIglesias) [inline scss/footer.scss]: `padding: 45px 25px 15px;`

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

**Comment by jeymarsonwolox** [comment]
Correction in padding footer

File: index.html [hunk 24]
```diff
@@ -1,14 +1,90 @@
...
   <title>HTML & CSS Training</title>
   <meta name="description" content="training">
   <meta name="author" content="Wolox">
-  <link rel="stylesheet"  type="text/css" href="application.css">
+  <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
+  <link rel="stylesheet" type="text/css" href="application.css">
 </head>
 <body>
-  <h1 class="title">Hello World</h1> 
+  <nav class="nav">
```
Comment (FrankIglesias) [inline index.html:12]: I think the wolox logo is missing

**Review by FrankIglesias — CHANGES_REQUESTED** [review]
One comment

**Comment by jeymarsonwolox** [comment]
Change wolox logo to img

**Review by FrankIglesias — APPROVED** [review]

**Pull request merged**